llvm-spirv/test/DebugInfo/X86/xray-split-dwarf-interaction.ll

```diff
@@ -11,7 +11,7 @@
 ; RUN:     -function-sections -data-sections \
 ; RUN:     -relocation-model=pic -filetype=obj \
 ; RUN:     -generate-type-units -o - %s | \
-; RUN:     llvm-readelf -sections | \
+; RUN:     llvm-readelf -sections - | \
```

Comment:
Can be removed like in https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/650/files